pkgs/by-name/po/powersploit/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "PowerSploit";
+  version = "v3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "PowerShellMafia";
+    repo = "PowerSploit";
+    rev = finalAttrs.version;
+    sha256 = "sha256-cFj4IwDyiwQDSYLCcloMQ83T8MOtMcKsyMilfrZOyHs=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/powersploit
+    cp -a --no-preserve=ownership * $out/share/powersploit
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/mattifestation/PowerSploit";
```

We don't know if the project is becoming active again. Then we have the changelog entry for tools (e.g., `nix-update`) which will add a link to the commit message.

```suggestion
    changelog = "https://github.com/PowerShellMafia/PowerSploit/releases/tag/v${version}";
```